src/commands.c

```diff
@@ -2240,22 +2242,22 @@ IMPLEMENT_COMMAND (event)
 
     ARG_CHECK (argv, 1);
 
-    gchar **split = g_strsplit (argv_idx (argv, 0), " ", 2);
-
-    const gchar *event = split[0];
-    const gchar *arg_string = split[1];
+    const gchar *event = argv_idx (argv, 0);
 
     if (event) {
         GString *event_name = g_string_ascii_up (g_string_new (event));
+        GArray *event_args = g_array_new (TRUE, FALSE, sizeof (gchar *));
+        g_array_insert_vals (event_args, 0,
+                             &g_array_index (argv, char *, 1),
+                             argv->len - 1);
 
         uzbl_events_send (USER_EVENT, event_name->str,
-            TYPE_FORMATTEDSTR, arg_string ? arg_string : "",
```

Comment:
Looks like it's also used in `uzbl_commands_send_builtin_event` but looks like a good candidate for replacing with TYPE_STR_ARRAY also.